pyomo/util/latex_printer.py

```diff
@@ -0,0 +1,619 @@
+import pyomo.environ as pe
```

Comment:
Following up on this, though, since you have a finite set of objects being imported, it's probably better for maintainability long-term to just import said objects instead of all of `pyomo.environ`.

Reply:
Fixed in coming commit

Reply:
I am rejecting the direct import of objects to keep explicit namespace clarity.  If someone feels strong enough to overrule me, I could be convinced.